README.adoc

```diff
@@ -914,13 +915,13 @@ Always use https://github.com/travisjeffery/timecop[Timecop] instead of stubbing
 it 'offsets the time 2 days into the future' do
   current_time = Time.now
```

Comment:
Judging by the docstring, this should be `Time.now + 2.days` or `2.days.from_now`.

Reply:
The return value of the method under test is 2 days into the future - what we're doing here is stubbing the time to be the current time.